src/main/resources/i18n/messages.properties

```diff
@@ -119,10 +119,10 @@ FoldTab.64=<html><strong>Default:</strong> Enabled if filename prettifying is en
 FoldTab.65=Monitor played status of files
 FoldTab.66=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> Folders must have "Monitor played status of files" enabled to have their contents included in the New Media folder.</html>
 FoldTab.67=Do nothing
-FoldTab.68=Add an overlay to the thumbnail
-FoldTab.69=Hide the file
-FoldTab.70=Move the file to a different folder
-FoldTab.71=Move the file to the recycle/trash bin
+FoldTab.68=Tag media
```

Comment:
The word "file" seems better to me than "media". Also "tag" has another common meaning that isn't what we mean, so I don't like that word either.